configs/chakra-ui.json

```diff
@@ -5,12 +5,15 @@
   ],
   "stop_urls": [],
   "selectors": {
-
-    "lvl0": "main h1",
-    "lvl1": "main h2",
-    "lvl2": "main h3",
-    "lvl3": "main h4",
-    "lvl4": "main h5",
+    "lvl0": {
+      "global": true,
+      "default_value": "Documentation",
+      "selector": "main h1"
+    },
+    "lvl1": "main h2 .content",
+    "lvl2": "main h3 .content",
+    "lvl3": "main h4 .content",
+    "lvl4": "main h5 .content",
```

Comment:
```suggestion
    "lvl0": {
      "global": true,
      "default_value": "Documentation",
      "selector": "main h1"
    },
    "lvl1": "main h2",
    "lvl2": "main h3",
    "lvl3": "main h4",
    "lvl4": "main h5",
```

Loooking at your website, this should be enough

Reply:
Yes, I tested this out locally.

It worked but also had some weird css classnames from the `<a>` tag, I think this is caused by emotion since we use CSS-in-JS. I thought it might be better to provide better specificity by wrapping the main content in a `span`. You get what I mean?

Looking at the structure of our MDX components, It might be 
<img width="754" alt="Screen Shot 2021-01-26 at 5 44 01 PM" src="https://user-images.githubusercontent.com/6916170/105852572-16930d80-5ffe-11eb-8205-cc062291c3e5.png">

Reply:
Oh I see, with emotion you should also add some client side rendering, I'll push an update on the PR that should solve your issue

Reply:
Oh, that would be awesome. Thank you 🙏🏽

Reply:
https://github.com/algolia/docsearch-configs/pull/3341/commits/443c123a5f0be9042c60de68b353fa5c8e0f0af9 should fix it!